src/screens/TripDetails/components/TripSection.tsx

```diff
@@ -141,6 +142,15 @@ const TripSection: React.FC<TripSectionProps> = ({
             <SituationMessages mode="no-icon" situations={leg.situations} />
           </TripRow>
         )}
+        {leg.transportSubmode === TransportSubmode.RailReplacementBus && (
+          <TripRow rowLabel={<ThemeIcon svg={Warning} />}>
+            <MessageBox type="warning" icon={null}>
+              <ThemeText>
+                {t(TripDetailsTexts.messages.departureIsRailReplacementBus)}
+              </ThemeText>
+            </MessageBox>
```

Comment:
I think this should use `message` param instead of `children` to get the styling correct. Is the image in the PR from the actual running app? 🤔 Since I think this will give white text in dark mode.

Reply:
Good catch, had not commited line 149 for some reason, you're absolutely correct. Anyways, I will update to using the message param. 

<img width="610" alt="image" src="https://user-images.githubusercontent.com/21310942/192769387-526e47bd-e77c-4518-8264-a25ae4fa6b18.png">